index.d.ts

```diff
@@ -2524,14 +2525,46 @@ declare namespace Eris {
   }
 
   export class VoiceConnection extends EventEmitter implements SimpleJSON {
-    channelID: string;
+    bitrate: number;
+    channelID: string | null;
+    channels: number;
     connecting: boolean;
-    current?: VoiceStreamCurrent;
+    connectionTimeout: NodeJS.Timeout | null;
+    current?: VoiceStreamCurrent | null;
+    ended?: boolean;
+    endpoint: URL;
+    frameDuration: number;
+    frameSize: number;
+    heartbeatInterval: NodeJS.Timeout | null;
     id: string;
+    mode?: string;
+    modes?: string;
+    opus: { [s: string]: unknown }; // TODO
```

Comment:
.

Reply:
`userID`:  `OpusScript` from `opusscript` | `OpusEncoder` from `@discordjs/opus`

Both are optional packages so not sure if this can be typed? I'm perfectly fine with leaving this untyped/`unknown` anyway

Reply:
Leaving as unknown seems to be the best option without requiring a hard install then